pravega/pravega-operator
# Issue522: Re-evaluate log level for pravega operator

### Change log description
Adding code changes to set the correct log level for various logs messages in pravega operator code 

### Purpose of the change
Fixes #522 

### What the code does
Using Zerolog setting various log messages with correct log level

### How to verify it
Deploy operator and pravega cluster with various loglevels set and check if corresponding logs are displayed or not.
